scripts/generator/graphql_generator/csharp/Editor/ShopifyDependencies.cs.erb

```diff
@@ -0,0 +1,128 @@
+// <copyright file="ShopifyDependencies.cs" company="Google Inc.">
```

Comment:
This file is a template from the PlayServicesResolver library with changes to which dependencies we use.